src/thunderbots/software/ai/navigator/util.h

```diff
@@ -18,3 +19,14 @@
  */
 double calculateTransitionSpeedBetweenSegments(const Point &p1, const Point &p2,
                                                const Point &p3, double final_speed);
+
+/**
+ * Takes a vector of points that represents a path and converts them to a vector of move
+ * primitives
+ *
+ * @param robot_id the robot_id to assign the move primitives and path to
+ * @param points the vector of points representing a path
+ * @return a vector of move primitives converted from a vector of points
+ */
+std::vector<MovePrimitive> convertToMovePrimitives(unsigned int robot_id,
```

Comment:
rename this to convertPointsToMovePrimitives

Reply:
renamed

Reply:
is this changed? I'm not seeing the new name?